package/src/bin/omicron-package.rs

```diff
@@ -265,12 +265,21 @@ async fn do_target(
         format!("failed to create directory {}", target_dir)
     })?;
     match subcommand {
-        TargetCommand::Create { image, machine, switch, rack_topology } => {
+        TargetCommand::Create {
+            image,
+            machine,
+            switch,
+            rack_topology,
+            clickhouse_topology,
+        } => {
             let target = KnownTarget::new(
                 image.clone(),
                 machine.clone(),
                 switch.clone(),
                 rack_topology.clone(),
+                clickhouse_topology
+                    .clone()
+                    .expect("clickhouse topology should never be NONE"),
```

Comment:
Why is this an `Option` if it's never allowed to be `None`? Maybe we want to use a default value in the `clap` attributes?

Reply:
😅 lolwat? Not sure what I was going for here as I did set the default to single-node https://github.com/oxidecomputer/omicron/blob/ecc95decfb09638fba2641a6d0f826ff7ae83fe4/package/src/lib.rs#L75